### ISSUE TITLE
Links in documentation reloads the whole page

### ISSUE THREAD

**Issue by slifty** [tvkitchen/tv-kitchen-website#33]
#### Current Behavior
When I click the documentation nav it only refreshes the content of the main space of the page, but if I embed a link to another documentation page (or another page on the site) I get a full "blank" refresh.

#### Input
On a documentation page I have:

```
Check out the [Quick Start](quick-start) guide if you want to create your own TV Kitchen.
```

(and a separate documentation page with the slug `quick-start`)

#### Expected Behavior
It would be ideal if it could load those pages in the same way that nav loads, where only the relevant component is updated instead of re-loading the entire page.

#### Possible Solutions

Is it possible to detect relative links and use whatever loading mechanism creates the nice load instead?

**Comment by piotrfonte** [comment]
@slifty could I ask you to test this now and report back, please?

**Comment by slifty** [comment]
I'm still seeing it unfortunately.

I imagine this will be easier for you to fix once I have #30 complete.

**Comment by piotrfonte** [comment]
Probably fixed in 890d274edf3b69be021be9c7195994ec8401fb76. Please check.
